std/datetime.d

```diff
@@ -6169,6 +6177,9 @@ public:
         $(TR $(TD SysTime) $(TD -) $(TD duration) $(TD -->) $(TD SysTime))
         )
 
+				This is DST-aware. As such, adding an hour to a given SysTime will sometimes not result in
+				an apparent change of time.
```

Comment:
Saying that it's DST-aware is wrong. opBinary ignores the timezone completely. It does all calculations in UTC. It happens that the time before and after adding or removing a duration could result in the same time in another time zone if that time has DST, but the UTC times are different, and opBinary doesn't care one whit about the time zone, because it's only dealing with the time in UTC - and the documentation on the top of `SysTime` should make it quite clear that `SysTime` holds the time internally in UTC and _not_ in the local time.